openreview/venue/group.py

```diff
@@ -172,6 +174,16 @@ def create_venue_group(self):
             'desk_rejection_name': { 'value': 'Desk_Rejection'}
         }
 
+        if self.venue.submission_stage.submission_email:
+            email_template = self.venue.submission_stage.submission_email
+            email_template = email_template.replace('{{Abbreviated_Venue_Name}}', self.venue.get_short_name())
```

Comment:
what about doing this replacement in the process function directly so we replace the values only once at the moment of sending the email.